scripts/linters/css_linter.py

```diff
@@ -58,6 +59,36 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    @staticmethod
+    def _get_trimmed_error_messages(lint_messages):
+        """Remove extra bits from stylelint error messages.
+
+        Args:
+            lint_messages: str. Messages returned by the css linter.
+
+        Returns:
+            str. A string with the trimmed error messages.
+        """
+        error_messages = ''
+        # We need to extract messages from the list and split them line by
+        # line so we can loop through them.
+        messages = lint_messages.split('\n')
+        for message in messages:
+            # Stylelint messages start with line numbers and then a
+            # cross(x) and a message-id in the end. We are matching
```

Comment:
By cross(x) do you mean the single character "x"?

If so, just say that (`"x"`).